pyvrp/cpp/educate/LocalSearch.cpp

```diff
@@ -212,10 +212,8 @@ void LocalSearch::loadIndividual(Individual const &individual)
 
         startDepot->tw = clients[0].tw;
         startDepot->twBefore = clients[0].tw;
-        startDepot->twAfter = clients[0].tw;
 
         endDepot->tw = clients[0].tw;
-        endDepot->twBefore = clients[0].tw;
         endDepot->twAfter = clients[0].tw;
```

Comment:
Question: do we need `startDepot->twBefore` and `endDepot->twAfter`?

Reply:
Those values should always be equal to `depot->tw` and never change, and are independent of the individual or the route, so I think the best place would be in the constructor of local search (as the instance cannot change), not `loadIndividual` or `Route::update`. However we can treat that separate of this bugfix (if you agree)?